Makefile

```diff
@@ -38,7 +38,7 @@ lint:
 	)
 
 test:
-	python -m pytest tests
+	python -m pytest -n auto tests
```

Comment:
This actually enables tests to run in parallel, reducing the testing time multifold.